.gitignore

```
@@ -3,5 +3,5 @@ worlds/.*.wbproj
 build/
 
 /controllers/nugus_controller/nugus_controller
-
-/nugus_controller_test
+/controllers/teleport_controller/teleport_controller
+/nugus_controller_test
```

Add newline at end of file to appease the gods
```suggestion
/nugus_controller_test

```